website/integrations: adds termix (#16889)
goauthentik/authentik: website/integrations/infrastructure/termix/index.mdx

    - **Application**: provide a descriptive name, an optional group for the type of application, the policy engine mode, and optional UI settings.
    - **Choose a Provider type**: select **OAuth2/OpenID Connect** as the provider type.
    - **Configure the Provider**: provide a name (or accept the auto-provided name), the authorization flow to use for this provider, and the following required configurations.
        - Note the **Client ID**, **Client Secret** and **slug** values because they will be required later.
        - Set a `Strict` redirect URI to `https://termix.company/users/oidc/callback`.
        - Select any available signing key.
    - **Configure Bindings** _(optional)_: you can create a [binding](/docs/add-secure-apps/flows-stages/bindings/) (policy, group, or user) to manage the listing and access to applications on a user's **My applications** page.

3. Click **Submit** to save the new application and provider.

## Termix configuration

> 1. Login to Termix as an administrator.

Reviewer: ```suggestion
1. Log in to Termix as an administrator.
```

> 2. Click on your admin username at the bottom left and open the **Admin Settings** interface.

Reviewer: ```suggestion
2. Click your admin username at the bottom left and open the **Admin Settings** interface.
```

3. Navigate to the **OIDC** tab.
4. Set the following required settings:
    - **Client ID**: Client ID from authentik

>     - **Client Secret**: Client Secrert from authentik

Reviewer: ```suggestion
    - **Client Secret**: Client secret from authentik
```

>     - **Authorization URL**: `http://authentik.company/application/o/authorize/`

Reviewer: no https?

    - **Issuer URL**: `http://authentik.company/application/o/<application_slug>/`
    - **Token URL**: `http://authentik.company/application/o/token/`
5. Click **Save Configuration**.

## Configuration verification

> To verify that authentik is correctly integrated with Termix, first log out of your account. Then, click **External** followed by **Login with External Provider**. You should be redirected to your authentik instance and after successfully authenticating, you should be redirected to Termix and logged in.

Reviewer: ```suggestion
To verify that authentik is correctly integrated with Termix, first log out of your Termix account. From the login screen, select **External** and then choose **Login with External Provider**. You should be redirected to your authentik instance, and after successfully authenticating, the system will return you to Termix where you will be logged in automatically.
```

    - **Application**: provide a descriptive name, an optional group for the type of application, the policy engine mode, and optional UI settings.
    - **Choose a Provider type**: select **OAuth2/OpenID Connect** as the provider type.
    - **Configure the Provider**: provide a name (or accept the auto-provided name), the authorization flow to use for this provider, and the following required configurations.

>         - Note the **Client ID**, **Client Secret**, and **slug** values because they will be required later.

Reviewer: good catch on that
